src/DataRenderer.tsx

```diff
@@ -17,7 +17,8 @@ export interface StyleProps {
   expandIcon: string;
   collapseIcon: string;
   collapsedContent: string;
-  noQuotesForStringValues: boolean;
+  noQuotesForStringValues?: boolean;
+  quotesForFieldNames?: boolean;
```

Comment:
@AnyRoad 
I updated this PR to make field name quotes optional via `quotesForFieldNames` style config.

Reply:
Thank you! 👍